internalshared/configutil/listener.go

```diff
@@ -123,6 +123,14 @@ type Listener struct {
 	// ChrootNamespace will prepend the specified namespace to requests
 	ChrootNamespaceRaw interface{} `hcl:"chroot_namespace"`
 	ChrootNamespace    string      `hcl:"-"`
+
+	// Per-listener redaction configuration
```

Comment:
Do we expect to expose the `withRedactionValue` config to customers at all? This is more for my curiosity than a blocking comment for the PR or anything!

Reply:
I don't think it's planned to be configurable by the end user at the moment, it was more so that as developers we had a way to change it (and tests that ran on its value), maybe it could have been done in a simpler way though. 🤔

Reply:
That makes sense! Honestly, on second thought, it would be weird if we did offer it, they could maybe lie about what version they were running to API clients!

Appreciate the response! :)